charts/athens-proxy/README.md

```diff
@@ -0,0 +1,150 @@
+# Athens Proxy Helm Chart
+
+## What is Athens?
+
+[Athens](https://docs.gomods.io) is a Server for Your Go Packages.
+
+Athens provides a server for [Go Modules](https://github.com/golang/go/wiki/Modules) that you can run. It serves public code and your private code for you, so you don't have to pull directly from a version control system (VCS) like GitHub or GitLab.
+
+## Prerequisites Details
+
+* Kubernetes 1.10+
+
+## Requirements
+
+- A running Kubernetes cluster
+- [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) installed and setup to use the cluster
+- [Helm](https://helm.sh/) installed and setup to use the cluster (helm init) or [Tilerless Helm](https://github.com/rimusz/helm-tiller)
```

Comment:
Two requests here:

- Can you replace `Tilerless` with `Tillerless`?
- Can you include links to documentation on how to install helm? Your tillerless helm documentation looks great fwiw 😁 💯